datascience-notebook/Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN npm install jupyterlab-flake8 \
     && jupyter labextension install jupyterlab-flake8 \
     && jupyter labextension install @ryantam626/jupyterlab_code_formatter \
     && jupyter serverextension enable --py jupyterlab_code_formatter
+    && pip install --upgrade nbstripout 
```

Comment:
I think this may break because of the missing `\` in the line above. Don't know if you tried to build this image.

Also, any reason for the `--upgrade` option if this is a newly installed package?

Reply:
No you don't need to add / because it's the last line
I guess because if they upgrade nbstripout which probably will happen, then when you build the docker image every time then it should update

Reply:
sorry what I meant (and I couldn't select the previous line on GitHub) is this is adding to the greater `RUN` command starting above but the previous line doesn't end with `\` which means `docker build` should complain (*I think...*)

Reply:
Regarding the `--upgrade`, I see that `nbstripout`'s documentation have it and as I said it shouldn't harm. 
But Docker is relatively "dumb" when it comes to decide whether it will reuse or not a cached layer. It would basically just compare the string.

Essentially if the line says `RUN pip install --upgrade foo` and you re-build the image several times, he would reuse the cached one even if a new version of the `foo` package is available (although I've seen options to make `docker build` ignore cache, etc...)